CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+cmake_minimum_required(VERSION 3.7)
```

Comment:
Can you change this to work with cmake 2.8? Ubuntu 14.04 will be supported until 2019-04 and has cmake 2.8.